lib/main.ts

```diff
@@ -305,6 +312,22 @@ export class Transpiler {
     this.visit(expr);
   }
 
+  static IGNORED_ANNOTATIONS = {'CONST': true, 'ABSTRACT': true, 'IMPLEMENTS': true};
+
+  isEmptyImport(n: ts.ImportDeclaration): boolean {
+    var bindings = n.importClause.namedBindings;
+    if (bindings.kind != ts.SyntaxKind.NamedImports) return false;
+    return (<ts.NamedImports> bindings).elements.every((e) => {
+      return Transpiler.IGNORED_ANNOTATIONS.hasOwnProperty(this.ident(e.name));
+    });
+  }
+
+  filterImports(ns: ts.ImportOrExportSpecifier[]) {
```

Comment:
it's a little scary to have this unused code, which may rot before we get around to re-enabling the feature.
How about making the predicate a static field?
```static isIgnoredAnnotation = (e) => {...}```
 then it could be used at line 321,
```elements.every(isIgnoredAnnotation)```
 and this filter method could be inlined below?